ckan/templates/user/edit_user_form.html

```diff
@@ -1,11 +1,10 @@
 {% import 'macros/form.html' as form %}
 
-<form class="dataset-form form-horizontal" method="post" action="{{ action }}">
+<form id="user-edit" class="dataset-form form-horizontal" method="post" action="{{ action }}">
```

Comment:
@joetsoi recently submitted [a pull request that adds ids to form tags](https://github.com/ckan/ckan/pull/2330/files) for testing purposes. This should probably follow the same naming convention, i.e. the id should be `user-edit-form`.